src/Swiping/SwipeProfiles.js

```diff
@@ -213,56 +163,51 @@ class SwipeProfiles extends React.Component {
       this.getPotentialFriendList();
     }
 
-    
-
     return (
-      /*navigation bar and other necessary information about the match*/
       <div className="SwipeProfile">
+        <Header id={this.state.id} />
         <br></br>
         <header class="pageTitle">Potential Friends!</header>
         <br></br>
-            <Header id={id}/>
+
+        <br></br>
+        <br></br>
+        <br></br>
+        {error ? (
+          <text>{error}</text>
+        ) : (
+          <div>
             <img src="ppFriendsLogo.png"></img>
-            <br></br>
-            <br></br>
-            <br></br>
-            {error ? (
-              <text>{error}</text>
-            ) : (
-              <div>
-                <p>Name: </p>
-                <text>{this.state.firstName}</text>
-                <p>Age: </p>
-                <text>{this.state.age}</text>
-                <br></br>
-                <div class="profileIntroSection">
+            <h1>A potential Friend!</h1> 
+           <div class = "row">
+              <div class="column left">
+                <div class = "profileLeft">
+                  <img src = "profilepic.png" alt = "profilepic" width="150"></img>
+                  <h1>{this.state.firstName}, ({this.state.age})</h1>
+                  <button class="button letsTalkButton" onClick={() => this.handleSwipe(true)}>
+                    Let's Talk
+                  </button>
                   <br></br>
-                  <p>Gender: </p>
+                  <button class="button notInterestedButton" onClick={() => this.handleSwipe(false)}>
+                      Not Interested
+                  </button>
+                </div>                
+              </div>
+              <div class="column right">
+                <div class="profileIntroSection">
+                  <p>Gender 👫 </p>
                   <text>{this.state.gender}</text>
-                  <p>Description: </p>
+                  <p>Description 😶 </p>
                   <text>{this.state.description}</text>
-                  <p>Interests: </p>
+                  <p>Interests 🎨 </p>
                   <text>{this.state.interests}</text>
-                  <p>Education / Work: </p>
+                  <p>Education / Work 💻 </p>
                   <text>{this.state.workplace}</text>
```

Comment:
are you going to add a section for the person's bio? or should we leave it out?

Reply:
oh wait nevermind, you have it as description. Should we change the naming to bio instead?